content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -80,7 +80,7 @@ The project containing the sample CorDapp opens.
 `corda-cli network config docker-compose solar-system`
 
 3. Deploy your network by building the network deployment Docker file using Corda CLI:
-  `corda-cli network deploy -n solar-system -f solar-system.yaml > solar-system-compose`
+  `corda-cli network deploy -n solar-system -f solar-system.yaml > docker-compose.yaml`
```

Comment:
I think this change conflicts with the change in https://github.com/corda/corda-docs-portal/pull/275